From 29df2bd0bc9f4151f5f914b9b46b76beae117e94 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Wed, 21 Feb 2018 14:13:26 +0100
Subject: [PATCH 2/8] Fix build without AppStream

Don't try to link to AppStream-Qt if building with APPSTREAM=OFF
Differential Revision: https://phabricator.kde.org/D10707
---
 libapper/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libapper/CMakeLists.txt b/libapper/CMakeLists.txt
index 6581778..485e91d 100644
--- a/libapper/CMakeLists.txt
+++ b/libapper/CMakeLists.txt
@@ -59,7 +59,10 @@ target_link_libraries(apper_private
     KF5::I18n
     Qt5::Core
     PK::packagekitqt5
-    AppStreamQt
 )
 
+if(APPSTREAM)
+    target_link_libraries(apper_private AppStreamQt)
+endif()
+
 install(TARGETS apper_private DESTINATION ${CMAKE_INSTALL_LIBDIR}/apper)
-- 
2.17.1

